README.md

```diff
@@ -1,3 +1,101 @@
-# ApplicationAssessment
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-[The Pragmatic checklist to assess an application](./assessment-checklist.md)
+# The Pragmatic checklist to assess an application
+
+When you plan to modernize an application, there are several questions to ask yourself/your customer to apprehend when the modernization would bring value.
+
+This is a draft of potential questions you should take into consideration when doing an assessment.
+"
+The Excel file can be downloaded [here](Application-Assessment.xlsx).
```

Comment:
Prefer make the whole sentence clicable, like `[The Excel file template can be downloaded.](Application-Assessment.xlsx)`.